### PR TITLE
Updated deps, fixed warnings, fixed small bug

### PR DESCRIPTION
**Pull request by DasEtwas**
fixed all clippy and cargo warnings, updated dependencies (fixed noise seed in demo, removed unneeded image dependency), satisfied cargo-audit and fixed small bug in Planet shape (min_radius)

### CODE REVIEW

File: examples/demo/planet.rs [hunk 1]
```diff
@@ -16,6 +16,7 @@ impl Planet {
     pub fn new() -> Self {
         Self {
             noise: noise::Fbm::new()
+                .set_seed(1337 * 1337)
```
Comment (Ralith) [inline examples/demo/planet.rs]: Was this included intentionally?

File: Cargo.toml [hunk 2]
```diff
@@ -22,15 +22,15 @@ ash = { version = "0.29", optional = true }
...
 
 [dev-dependencies]
 winit = "0.18.1"
 criterion = "0.2.10"
 vk-shader-macros = "0.2"
 memoffset = "0.2.1"
-half = "1.3.0"
-noise = "0.5.1"
+half = "1.4.0"
+noise = { version = "0.6.0", features = [] }
```
Comment (Ralith) [inline Cargo.toml]: Was this intended to be `default-features = false`?

File: examples/demo/planet.rs (hunk 1 above)
Comment (DasEtwas) [reply examples/demo/planet.rs]: As the `noise` crate dependency changed, it produced different noise, so I tried different seeds until there was terrain under the camera.

File: Cargo.toml (hunk 2 above)
Comment (DasEtwas) [reply Cargo.toml]: Yes, this is an error.

Comment (DasEtwas) [reply Cargo.toml]: fixed in commit
https://github.com/Ralith/planetmap/pull/2/commits/a6b58c7abb7805f4bbac7e58279bb894074248e9

**Pull request merged**